python/lsst/rubintv/production/utils.py

```diff
@@ -490,6 +492,32 @@ def catchPrintOutput(functionToCall, *args, **kwargs):
     return f.getvalue()
 
 
+def sanitizeNans(obj):
+    """Recursively sanitize an object of any NaN-valued items.
+
+    Nans are not JSON de-serializable, so this function replaces them with
+    ``None``.
+
+    Parameters
+    ----------
+    obj : `object`
+        Any object to sanitize.
```

Comment:
Maybe add a comment clarifying that this expects a `list` or `dict` or  individual object, but not any iterable?